### DISCUSSION TITLE
YupResolver in react-native not working

### DISCUSSION THREAD

**Discussion by acollazomayer** [react-hook-form/react-hook-form#6376] [Q&A]
I cant manage to make the yup resolver work in react-native. The formState has no errors. I don't know if I am missing something. But I am following the documentation correctly. Even if I force to return an object with errors nothing is inside the formState

**Comment by bluebill1049** [comment]
you want to try your code in a codesandbox and post it here?

**Reply by acollazomayer** [reply]
I think it's an environment issue, not a code issue. I will try to create a sample repo and see if I can reproduce the error

**Reply by bluebill1049** [reply]
or maybe resolver version mismach as well?
